### DISCUSSION TITLE
Does anyone know if faster/slower audio changes WERs?

### DISCUSSION THREAD

**Discussion by SIlver--** [openai/whisper#2117] [Q&A]
I imagine at a certain point enough increase/decrease will be detrimental to accuracy and different voices will provide different results in terms of speed. But just thinking out loud that an increase of even just 10% speed would be a big boost in terms of transcribing audio speed. As well as reducing costs overall.

Thoughts?

**Comment by glangford** [comment]
Worth experimenting to see for yourself, I would think that the effect would vary based on speaker clarity, audio quality/noise, speedup and language. Note that the original Whisper announcement (2022) had a demo transcript of very high speed speech, see "Speed talking" under examples here. 

https://openai.com/research/whisper

**Comment by EtienneAb3d** [comment]
I did some experiments with this, with several stretching tools (see WhisperHallu code), and didn't get any improvements.
Whisper is quite insensible to the voice speed.
The only difference I got is that the resulting damage on the sound quality just produce a lower recognition quality.

**Comment by misutoneko** [comment]
Yeah I had the same idea [here](https://github.com/linto-ai/whisper-timestamped/discussions/128) a while back.
I didn't notice much difference in the transcript, but slowing down the audio did improve the timing of the subtitles slightly.

Also, at some point whisper.cpp had a switch that sped up the audio but I guess it's been dropped since.

**Comment by itaipee** [comment]
on most Kaldi recipes , speed-permutation was a default augmentation method , and usually performs well , gives you a "free"  1%-2% reduction in WER.    
However, Kaldi Acoustics models size is 10M-40M parameters , and are usually trained on 50-1000 hours of audio.   So Whisper large model is several magnitude larger than Kaldi , and was trained on ~1000X more audio.     I can assume the diversity of the audio is so much more , so speed augmentation becomes redundant.  

It might helps if you want to fine-tune to a very unique domain , but with smaller training set.
